### PR TITLE
MM-19477: model extra manifest data

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary 
Model `HomepageURL` and `SupportURL` for use in listing plugins in the marketplace.

#### Ticket Link
Fixes: https://mattermost.atlassian.net/browse/MM-19477

### CODE REVIEW

File: model/manifest.go [hunk 1]
```diff
@@ -130,6 +130,12 @@ type Manifest struct {
 	// A description of what your plugin is and does.
 	Description string `json:"description,omitempty" yaml:"description,omitempty"`
 
+	// HomepageURL is an optional link to learn more about the plugin.
+	HomepageURL string `json:"homepage_url,omitempty" yaml:"homepage_url,omitempty"`
+
+	// SupportURL is an optional URL where plugin issue can be reported.
```
Comment (hanzei) [inline model/manifest.go]: I'm stumbling across the term "plugin issues". What is a "plugin issue"? Why not just "issues"?

Comment (lieut-data) [reply model/manifest.go]:
```suggestion
	// SupportURL is an optional URL where plugin issues can be reported.
```

**Comment by prapti** [comment]
@lieut-data Can the [ticket](https://mattermost.atlassian.net/browse/MM-19477) be updated with QA test steps? That will be very helpful.

**Comment by lieut-data** [comment]
@prapti, done! Note that there's not much to test as part of this PR specifically: the changes to other repositories to use these new fields will provide the opportunity to verify same.

**Comment by hanzei** [comment]
I'm wondering if we should update the example in https://github.com/mattermost/mattermost-server/blob/82a826d3db6ec9be254f5b1063b486866c613bb4/model/manifest.go#L90-L120. Thoughts?

**Comment by hanzei** [comment]
@lieut-data :point_up:

**Comment by lieut-data** [comment]
Thanks for the poke, @hanzei -- submitted https://github.com/mattermost/mattermost-server/pull/13016.

**Pull request merged**